enocean/eep_parser.py

```diff
@@ -243,13 +253,13 @@ def _parse_eep_F6_02_03(self, payload, status):
         return results
 
     def _parse_eep_F6_10_00(self, payload, status):
-        #logger.debug("enocean: processing F6_10_00: Mechanical Handle")
+        logger.debug("enocean: processing F6_10_00: Mechanical Handle")
         results = {}
         if (payload[0] == 0xF0):
             results['STATUS'] = 0
         elif ((payload[0]) == 0xE0) or ((payload[0]) == 0xC0):
             results['STATUS'] = 1
-        # Typo error in older Eltako Datasheet for 0x0D instead of the right 0xD0
```

Comment:
Please revert your changes. There was in fact a typo (Schreibfehler) not a type errror.